aztarna/ros/ros2/helpers2.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+
+default_topics = ['/rosout', '/parameter_events']
+
+
+class ROS2host:
```

Comment:
Proposed another helpers file (to avoid messing with the previous one which is still used by the rclpy-based impl.). Your input here @LanderU is more than welcome. Frankly, we could also get things done (likely faster) with a different data structure (e.g. list of dicts).

I don't have a strong preference but think that keeping these abstractions here might actually help some future tasks (e.g. mixing together passive and active reconnaissance data). Let me know if you have another proposal here.